web/src/stores/languages.ts

```diff
@@ -22,11 +23,14 @@ enum ActionType {
 
 interface LoadedAction {
   type: ActionType.LOADED;
-  allLocales: Locales;
-  contributableLocales: Locales;
-  nativeNames: NativeNames;
-  rtlLocales: Locales;
-  translatedLocales: Locales;
+  payload: {
+    allLocales: Locales;
+    contributableLocales: Locales;
+    nativeNames: NativeNames;
+    rtlLocales: Locales;
+    translatedLocales: Locales;
+    contributableNativeNames: NativeNames;
+  };
 }
```

Comment:
Fixing this because Redux actions should follow the convention:

```
type ACTION = {
  action: ACTION_TYPE;
  payload: ACTION_PAYLOAD;
}
```

Reply:
@mozgzh I've tested this locally and it looks good. Also, the unit test for the input language component also passes

Reply:
awesome :rocket: